src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -211,7 +211,7 @@ bool QMCFixedSampleLinearOptimizeBatched::run()
 {
   if (do_output_matrices_csv_ && !output_matrices_initialized_)
   {
-    numParams = optTarget->getNumParams();
+    int numParams = optTarget->getNumParams();
```

Comment:
When copying a constant for more concise code, `const int` is preferred to prevent unexpected change of these parameters.